heartbeat/tomcat

```diff
@@ -56,6 +56,10 @@ else
   SU=su
 fi
 
+if which systemctl > /dev/null 2>&1; then
```

Comment:
systemd/systemctl could be installed and yet not used. Probably better to test for a running daemon.